settings.php

```diff
@@ -237,10 +237,13 @@
                                 <div class="box-body">
                                     <div class="row">
                                         <div class="col-lg-12">
-                                            <?php
-                                            $FTLpid = intval(pidofFTL());
+<?php
+// Try to get FTL PID
+$FTLpid = intval(pidofFTL($FTLrestarted));
```

Comment:
I'm not sure if I understand what your are doing here. Why are your passing `$FTLrestarted` to `pidofFTL()`. That function does not make use of passed argements.